weblate/trans/backups.py

```diff
@@ -630,6 +678,8 @@ def restore(self, project_name: str, project_slug: str, user: User, billing=None
                 Label(project=project, **entry) for entry in self.data["labels"]
             )
             self.labels_map = {label.name: label for label in labels}
+            if "categories" in self.data:
+                self.restore_categories(self.data["categories"], None)
```

Comment:
The type annotation doesn't allow `None` here.

Reply:
When the parameter of a function has a default value of None, the annotation automatically picks up that is optional.
![image](https://github.com/user-attachments/assets/553e76fb-7e31-4629-a95a-e875d3046f2d)
 If you prefer, I can make it explicit.

Reply:
Yes, please see feedback from mypy on https://github.com/WeblateOrg/weblate/pull/14531/files. It is not yet enforced because we're not yet there with the type annotations, but I want new code to pass.